staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/compile.go

```diff
@@ -84,6 +85,56 @@ func BuildRequestType() *apiservercel.DeclType {
 	))
 }
 
+// BuildNamespaceType generates a DeclType for Namespace.
+// Certain nested fields in Namespace (e.g. managedFields, ownerReferences etc.) are omitted in the generated DeclType
+// by design.
```

Comment:
If use the schemaResolver that the type checker uses, these SSA fields are already omitted.

Reply:
I've reconsidered this matter. I think it is not worthwhile to wire in the whole OpenAPI definitions only for one type. I am planning to compromise with a test to ensure this hand-crafted namespace schema matches that of OpenAPI. What do you think?

Reply:
Let's postponed the check a bit later. The handcrafted type looks consistent with `v1.Namespace`. We can find a way to sync this later in next release since there is no change of behavior on the user's side.

Reply:
> I've reconsidered this matter. I think it is not worthwhile to wire in the whole OpenAPI definitions only for one type. I am planning to compromise with a test to ensure this hand-crafted namespace schema matches that of OpenAPI. What do you think?

SGTM!